### PR TITLE
Start NEXT development

### PR DESCRIPTION
**Pull request by JanPietrzyk**
* replace humbug with infection
* remove composer lock
* remove php 5.6 compatibility
* remove php 7.0 compatibility
* remove php 7.1 compatibility

### CODE REVIEW

File: .php_cs [hunk 1]
```diff
@@ -1,11 +1,13 @@
 <?php
```
Comment (teiling88) [inline .php_cs:1]: strict types?

File: .php_cs [hunk 2]
```diff
@@ -1,11 +1,13 @@
...
-    ->level(Symfony\CS\FixerInterface::PSR2_LEVEL)
-    ->finder($finder)
-    ;
\ No newline at end of file
+return PhpCsFixer\Config::create()
+    ->setRules([
+        '@PSR2' => true,
+    ])
+    ->setFinder($finder)
+;
```
Comment (teiling88) [inline .php_cs:13]: Missing new line at the end of the file

File: infection.phar.asc [hunk 3]
```diff
@@ -0,0 +1,11 @@
+-----BEGIN PGP SIGNATURE-----
```
Comment (teiling88) [inline infection.phar.asc:1]: pls integrate phar.io/phive here 
or if you want I will create an pr for that :-)

**Review by teiling88 — CHANGES_REQUESTED** [review]

File: .php_cs (hunk 1 above)
Comment (JanPietrzyk) [reply .php_cs:1]: ok, so I should add this file to the fixer then... :)

Comment (teiling88) [reply .php_cs:1]: Yes :-)

**Pull request closed without merging**